### ISSUE TITLE
Find and Relace button name is not announced by screen reader

### ISSUE THREAD

**Issue by jswiderski** [ckeditor/ckeditor5#10248]
## 📝 Provide detailed reproduction steps (if any)

1.  Open https://ckeditor.com/docs/ckeditor5/latest/features/find-and-replace.html
2.  Enable screen reader like JAWS or NVDA
3.  Navigate to Find and Replace button 

### ✔️ Expected result

Button name is read

### ❌ Actual result

Button name is not read

### ❓ Possible solution

_If you have ideas, you can list them here. Otherwise, you can delete this section._

## 📃 Other details

*   Browser: Any
*   OS: Any
*   First affected CKEditor version: 29.0.0
*   Installed CKEditor plugins: Find and Replace

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.

**Comment by bbenjamin** [comment]
Can't reproduce. Screenshot is from NVDA/Chrome/Win 10. Also tried in Chrome/VoiceOver and it also worked.
<img width="833" alt="NVDA-find-replace" src="https://user-images.githubusercontent.com/1742345/169890986-73357217-9125-4273-9215-59ce62092f46.png">
